ush/global_cycle_driver.sh

```diff
@@ -93,8 +94,17 @@ for n in $(seq 1 $ntiles); do
     ln -fs $FIXgfs/orog/${CASE}/C${CRES}.mx${OCNRES}_oro_data.tile${n}.nc   $DATA/fnorog.00$n
   fi
 
-  if [[ "$DO_SNO_INC" == ".true." ]] ; then  
-        ln -fs $COMIN/$PDY.${cyc}0000.xainc.tile${n}.nc      $DATA/xainc.00$n
+  if [[ "$DO_SNO_INC_JEDI" == ".true." ]] ; then  
+        ln -fs $COMIN/$PDY.${cyc}0000.xainc.tile${n}.nc      $DATA/snow_xainc.00$n
+  fi
+
+  if [[ "$DO_SOI_INC_JEDI" == ".true." ]] ; then
+#        ln -fs $COMIN/soil_sfcincr_jedi.00$n $DATA/soil_xainc.00$n
```

Comment:
You are pointing to a home directory for soil_xainc.00$n

Reply:
As Kate helps us hosting the data in $COMIN, I have now deleted the line to link file from my home directory. Only retained the line to link $COMIN. Thanks!

Reply:
Thanks!